Dockerfile

```diff
@@ -25,6 +22,11 @@ RUN npm prune --production && \
 # FROM directive instructing base image to build upon
 FROM python:3.6 AS app
 
+# Sets the local timezone of the docker image
+ARG TZ
+ENV TZ ${TZ:-America/Detroit}
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
```

Comment:
what is the benefit of moving this `TZ` setting here?

Reply:
I think moving these up higher is actually wrong, I'll move it back down where it was.
https://medium.com/@esotericmeans/optimizing-your-dockerfile-dc4b7b527756